packages/extension-chakra-storefront/jest.config.js

```diff
@@ -53,12 +53,12 @@ module.exports = {
         createTestGlob('pages/product-list'),
         createTestGlob('components/offline-banner'),
         createTestGlob('components/offline-boundary'),
-        // createTestGlob('pages/login'),  // TODO: enable after Account page has been migrated
+        createTestGlob('pages/login'),
         createTestGlob('pages/login-redirect'),
         createTestGlob('pages/social-login-redirect'),
-        createTestGlob('hooks/cart'),
-        // createTestGlob('pages/registration'), // TODO: enable after Account page has been migrated
-        '<rootDir>/src/pages/checkout/partials/contact-info.test.js',
+        createTestGlob('hooks/'),
+        createTestGlob('pages/registration'),
+        // '<rootDir>/src/pages/checkout/partials/contact-info.test.js',
```

Comment:
Will enable this back when passwordless bug is fixed in https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2783 is merged. I will port the changes to here and fix tests